src/Stratis.Bitcoin/Consensus/ConsensusManagerBehavior.cs

```diff
@@ -64,6 +64,11 @@ public class ConsensusManagerBehavior : NetworkPeerBehavior
         /// <seealso cref="https://en.bitcoin.it/wiki/Protocol_documentation#getheaders"/>
         private const int MaxItemsPerHeadersMessage = 2000;
 
+        /// <summary>Maximum number of hashes that we will accept in a message from a peer.</summary>
+        /// <seealso cref="https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-August/016285.html"/>
+        /// <seealso cref="https://github.com/bitcoin/bitcoin/pull/13907"/>
+        public const int MaxLocatorSize = 101;
+
```

Comment:
I'm not sure if this is the right file to have this. I'd probably inside of the block locator class. There are other possibilities, but the specific behaviour does not look like a good place to have it

Reply:
If it was acceptable for `MaxItemsPerHeadersMessage` to be in this file then it seems to me that, at least, we are no worse off including `MaxLocatorSize`.

Reply:
It is because the difference between those two is that one is private and the other one is used elsewhere.

Reply:
Block locator class seems more correct indeed.